pages.de/android/am.md

```diff
@@ -0,0 +1,20 @@
+# am
+
+> Android Aktivitäten Manager.
```

Comment:
Is this a proper name? If so, we should leave it at "activity manager". If not, we might want to avoid the "Idioten Leerzeichen".

Reply:
I'd go with `Androids Aktivitäten-Manager` as you suggested before.

Reply:
Alright. `Android-Aktivitäten-Manager` might also work, btw… 🤷🏻‍♀️